tests/typed_search.test.ts

```diff
@@ -218,17 +221,19 @@ describe.each([
       cropLength: 6,
       attributesToHighlight: ['*'],
       filter: 'title = "Le Petit Prince"',
-      matches: true,
+      showMatchesPosition: true,
     })
     expect(response.hits.length === 1).toBeTruthy()
     expect(response.offset === 0).toBeTruthy()
     expect(response.limit === 5).toBeTruthy()
     expect(response).toHaveProperty('processingTimeMs', expect.any(Number))
     expect(response.query === 'prince').toBeTruthy()
     expect(response.hits[0]?.title === 'Le Petit Prince').toBeTruthy()
-    expect(response.hits[0]?._matchesInfo?.title?.[0]?.start === 9).toBeTruthy()
     expect(
-      response.hits[0]?._matchesInfo?.title?.[0]?.length === 6
+      response.hits[0]?._matchesPosition?.title?.[0]?.start === 9
+    ).toBeTruthy()
+    expect(
+      response.hits[0]?._matchesPosition?.title?.[0]?.length === 6
```

Comment:
Maybe you can put `response.hits[0]?._matchesPosition?.title?.[0]` inside a variable and reuse it :)

Reply:
I will consider these changer in another PR where I rework the tests! Thanks :)